EIPS/eip-1062.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 1062
+title: Formalize IPFS hash into ENS(Ethereum Name Service) resolver
+author: Phyrex Tsai <phyrex@portal.network>, Portal Network Team
+discussions-to: phyrex@portal.network
```

Comment:
Please supply a URL where people can discuss, such as a post on the ethereum-magicians forum.